app/lib/nomis_client/base.rb

```diff
@@ -25,7 +25,7 @@ def put(path, params = {})
       REFRESH_TOKEN_TIMEFRAME_IN_SECONDS = 5
 
       def token_request(method, path, params)
-        token.send(method, "#{ENV['NOMIS_API_PATH_PREFIX']}#{path}", params)
+        token.send(method, "#{ENV['NOMIS_PRISON_API_PATH_PREFIX']}#{path}", params)
```

Comment:
We've gone with completely new environment variables for prefixes and site urls so that we can very easily rollback our configuration and make sure devs don't suddenly find their ci/local development doesn't work anymore.